ecs-cli/modules/config/readwriter.go

```diff
@@ -95,65 +127,13 @@ func (rdwr *IniReadWriter) IsInitialized() (bool, error) {
 }
 
 // IsKeyPresent returns true if the input key is present in the input section
-func (rdwr *IniReadWriter) IsKeyPresent(section, key string) bool {
+func (rdwr *INIReadWriter) IsKeyPresent(section, key string) bool {
 	return rdwr.cfg.Section(section).HasKey(key)
 }
 
-// ReadFrom initializes the ini object from an existing ecs-cli config object.
-func (rdwr *IniReadWriter) ReadFrom(ecsConfig *CliConfig) error {
-	return rdwr.cfg.ReflectFrom(ecsConfig)
-}
-
-// Save saves the config to a config file.
-func (rdwr *IniReadWriter) Save(dest *Destination) error {
-	destMode := dest.Mode
-	err := os.MkdirAll(dest.Path, *destMode)
-	if err != nil {
-		return err
-	}
-
-	path := configPath(dest)
-
-	// If config file exists, set permissions first, because we may be writing creds.
-	if _, err := os.Stat(path); err == nil {
-		err = os.Chmod(path, configFileMode)
-		if err != nil {
-			logrus.Errorf("Unable to chmod %s to mode %s", path, configFileMode)
-			return err
-		}
-	}
-
-	// Open the file, optionally creating it with our desired permissions.
-	// This will let us pass it (as io.Writer) to go-ini but let us control the file.
-	configFile, err := os.OpenFile(path, os.O_WRONLY|os.O_CREATE, configFileMode)
-
-	// Truncate the file in case the earlier contents are longer than the new
-	// contents, so there will not be any trash at the end of the file
-	configFile.Truncate(0)
-
-	if err != nil {
-		logrus.Errorf("Unable to open/create %s with mode %s", path, configFileMode)
-		return err
-	}
-	defer configFile.Close()
-
-	_, err = rdwr.cfg.WriteTo(configFile)
-	if err != nil {
-		logrus.Errorf("Unable to write config to %s", path)
-		return err
-	}
-
-	return nil
-}
-
-func configPath(dest *Destination) string {
-	return filepath.Join(dest.Path, configFileName)
-}
-
-func newIniConfig(dest *Destination) (*ini.File, error) {
+func newINIConfig(dest *Destination) (*ini.File, error) {
 	iniCfg := ini.Empty()
-	path := configPath(dest)
-	logrus.Debugf("using config file: %s", path)
+	path := iniConfigPath(dest)
 	if _, err := os.Stat(path); err != nil {
 		// TODO: handle os.isnotexist(path) and other errors differently
```

Comment:
We should remove this and the debug log. If the ini path doesn't exist, we now throw the yaml error.